admin/class-nginx-helper-admin.php

```diff
@@ -717,6 +717,13 @@ public function purge_all() {
 				break;
 		}
 
+		/**
+		 * Fire an action after the cache has been purged whatever caching type is used'
+		 * 
+		 * @since 2.2.2
+		 */
+		if ( 'purge' === $action ) do_action( 'rt_nginx_helper_after_purge_all' );
```

Comment:
```php
if ( 'purge' === $action ) {
	
    /**
     * Fire an action after the entire cache has been purged whatever caching type is used.'
     * 
     * @since 2.2.2
     */
    do_action( 'rt_nginx_helper_after_purge_all' );

}
```
Can you please make this change?